system-tests/tests/src/test/java/org/eclipse/dataspaceconnector/system/tests/local/FileTransferLocalSimulation.java

```diff
@@ -44,7 +44,7 @@ public FileTransferLocalSimulation() {
                 .protocols(http
                         .baseUrl(CONSUMER_CONNECTOR_HOST))
                 .assertions(
-                        global().responseTime().max().lt(2000),
+                        global().responseTime().max().lt(5000),
```

Comment:
nit: small change but does not relate to excluding modules :) can ignore

Reply:
Thanks, fixed the PR description